digital_certificates.yaml

```diff
@@ -0,0 +1,574 @@
+  version: 2.0.0
+  license:
+    name: MIT
+    url: https://github.com/aziontech/azionapi-openapi/blob/main/LICENSE.md
+servers:
+  - description: Production
+    url: https://api.azionapi.net
+security:
+  - tokenAuth: [asdadasd]
+paths:
+  /digital_certificates:
+    get:
+      tags:
+        - Retrieve digital certificate list
+      summary: List all existing digital certificates and CSRs registered to your account
+      operationId: listDigitalCertificates
+      parameters:
+        - name: order_by
+          in: query
+          description: ID of certificate to return
+          required: false
+          schema:
+            type: string
+            enum: [name, id]
+            default: id
+        - name: sort
+          in: query
+          description: ID of certificate to return
+          required: false
+          schema:
+            type: string
+            enum: [asc, desc]
+            default: asc
+        - name: name
+          in: query
+          description: Searches certificate for name
+          required: false
+          schema:
+            type: string
+        - name: search
+          in: query
+          description: Searches for string in certificate name
+          required: false
+          schema:
+           type: 
+            string 
+
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC200List'
+        "400":
+          description: Bad Request
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC400'
+        "401":
+          description: Unauthorized
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC401'
+        "403":
+          description: Forbidden
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC403'
+        "404":
+          description: Not Found
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC404'
+        "406":
+          description: Not acceptable
+          content:
+            application/json; version=3:
+              schema:
+                $ref: '#/components/schemas/DC406'
+
+    post:
+      tags:
+          - Create digital certificate
+      summary: Create a new digital certificate
+      operationId: createCertificate
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                certificate:
+                  type: string
+                private_key:
+                  type: string
```

Comment:
**todo:** there are some missing properties

I'm missing certificate_type, please check if there's any other missing property.

Reply:
The body of the POST request only contains those 3 properties. The response to that request may contain additional properties as certificate_type or id

Reply:
May we are missing something, because `certificate_type` should exist in the payload, this property allow to create "trusted_ca_certificate"